### ISSUE TITLE
requirements is requesting incompatible version of setuptools-scm

### ISSUE THREAD

**Issue by gkiar** [yeatmanlab/pyAFQ#213]
When installing via `pip install -e .` I get the following error. Updating `requirements.txt` to require `setuptools-scm[toml]=3.4.3` instead of `3.3.3` solves the issue. Before opening a PR, is there any reason to stick to `3.3.3`?

```
$ pyAFQ
Traceback (most recent call last):
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2742, in requires
    deps.extend(dm[safe_extra(ext)])
KeyError: 'toml'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/greg/code/env/auto/bin/pyAFQ", line 4, in <module>
    __import__('pkg_resources').require('pyAFQ==0.1.2.dev18+g18896cb.d20200207')
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3252, in <module>
    @_call_aside
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3236, in _call_aside
    f(*args, **kwargs)
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3265, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 584, in _build_master
    ws.require(__requires__)
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 901, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 795, in resolve
    new_requirements = dist.requires(req.extras)[::-1]
  File "/Users/greg/code/env/auto/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2745, in requires
    "%s has no such extra feature %r" % (self, ext)
pkg_resources.UnknownExtra: setuptools-scm 3.3.3 has no such extra feature 'toml'
```

**Comment by arokem** [comment]
No reason, but I am confused why the CI doesn't catch this. Either way, the fix in #206 seems good!

**Comment by arokem** [comment]
This is closed through the fix in e6fb420
